src/test/java/com/rapid7/util/SocketChannelReceiver.java

```diff
@@ -119,7 +120,7 @@ private void listenForMessages(Socket socket) throws IOException {
             if (messageLength == -1) {
                 continue;
             }
-            String[] messages = new String(Arrays.copyOf(buffer, messageLength)).split("\\r?\\n");
+            String[] messages = new String(Arrays.copyOf(buffer, messageLength), UTF_8).split("\\r?\\n");
```

Comment:
can this throw a `UnsupportedEncodingException`?

Reply:
Only the `String` constructor that receives the `charsetName` parameter as a `String` throws an `UnsupportedEncodingException` but here I'm using the one which receives a `Charset` from `StandardCharsets` which doesn't.